docs/UserGuide.md

```diff
@@ -67,6 +67,10 @@ and flags(e.g. `-p`, `-D`, etc.) are **case-sensitive**.
 Adds an item to the list of tracked inventory.
 
 Format: `addi n/ITEM_NAME q/QUANTITY d/DESCRIPTION [t/TAG]…​ sp/SELL_PRICE cp/COST_PRICE`
+- `SELL_PRICE` is the price that the user sells the item for
+- `COST_PRICE` is the price that the user purchased the item for
+- `SELL_PRICE` and `COST_PRICE` should be given as a number rounded to the nearest cent
+- TrackO allows users to input a `COST_PRICE` that is greater than the `SELL_PRICE` although user will be selling at a loss
```

Comment:
Sounds a bit informal. Maybe something like

TrackO allows items to have a larger `COST_PRICE` than `SELL_PRICE`, where items can be sold at a loss